_styles/main.css

```diff
@@ -288,16 +288,70 @@ nav a:visited {
 }
 
 nav a:focus {
-    color: #08c;
-    fill: #08c;
     text-decoration: none;
 }
 
-nav a:hover,
-nav a:visited:hover {
+nav a:visited{
     opacity: 0.7;
 }
 
+/*color changes of .nav icon on focus*/ 
+
+nav .right li a[title="Facebook"]:focus{
```

Comment:
This selector is over qualified. I would trim it down to `nav a[title="Facebook']:focus`